v2/src/components/VehicleCard.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { CarouselCardProps } from '../Interfaces';
+
+const totalDelay = (minutes: number) => {
+  const hours = Math.floor(minutes / 60);
+  let finalTime = '';
+
+  if (hours === 1) {
+    finalTime += `${hours} hour`;
+  } else if (hours > 1) {
+    finalTime += `${hours} hours`;
+  }
+
+  if (hours === 0) {
+    finalTime += `${minutes} minutes`
+  } else {
+    finalTime += `, ${minutes % 60} minutes`
+  }
+
+  return finalTime;
+}
+
+export const VehicleCard = (props: CarouselCardProps) => (
+  <div className="vehicle-card">
+    <div className="vehicle-card-header-container">
+      <div className={`transit-icon-container ${props.type}`}>
+        <img className="transit-icon" src={props.icon} alt="current vehicle icon" />
+      </div>
+      <h3 className="vehicle-card-header">
+        <b>Route {props.route}</b>
+      </h3>
+    </div>
+    <hr className="side-menu-divider" />
+    <p>
+      {props.direction} -> <b>{props.terminal}</b>
```

Comment:
suggestion: http://xahlee.info/comp/unicode_arrows.html

Reply:
Good idea, I made the ones we have currently but if its more efficient let's use this. Thoughts?